game/src/sandbox/dashboards/mod.rs

```diff
@@ -15,6 +15,7 @@ mod risks;
 mod selector;
 mod traffic_signals;
 mod travel_times;
+mod trip_problems;
```

Comment:
extracted all the "safety matrix" stuff into trip problems since it's shared by other modules now, and not just about "safety"